cleverhans/utils.py

```diff
@@ -111,7 +111,8 @@ def random_targets(gt, nb_classes):
              encoded as one-hot labels.
     """
     # If the ground truth labels are encoded as one-hot, convert to labels.
-    if len(gt.shape) == 2:
+    from keras.utils import np_utils
```

Comment:
Could you revert these lines? There was probably a small error when rebasing after #192 got merged